models/points.go

```diff
@@ -926,6 +933,22 @@ func scanNumber(buf []byte, i int) (int, error) {
 				return i, fmt.Errorf("unable to parse integer %s: %s", buf[start:i-1], err)
 			}
 		}
+	} else if isUnsigned {
+		// Make sure the last char is a 'u' for unsigned
+		if buf[i-1] != 'u' {
+			return i, ErrInvalidNumber
+		}
+		// Make sure the first char is not a '-' for unsigned
```

Comment:
This check is redundant with the initial check at line 833?

Reply:
I don't believe so. The check at line 833 prevents a person from writing `m=-`. This checks to make sure a negative wasn't used while looking at an unsigned integer.